domain/src/main/java/org/oppia/android/domain/exploration/ExplorationProgressController.kt

```diff
@@ -408,7 +412,11 @@ class ExplorationProgressController @Inject constructor(
                   commandQueue,
                   installationId,
                   learnerId,
-                  learnerAnalyticsLogger
+                  learnerAnalyticsLogger,
+                  oppiaClock.getCurrentTimeMs(),
+                  isContinueButtonAnimationSeen,
```

Comment:
It'd probably be cleaner to do the ``?: false`` here and avoid needing to propagate a nullable boolean.

Reply:
done